augur/application/db/models/augur_data.py

```diff
@@ -27,6 +27,8 @@
 import re
 import json
 import urllib.parse
+import pyLDAvis
+import numpy as np
```

Comment:
You seem to be importing numpy and PyLDAvis in a models file. Are these intended for performing business logic in the models? As far as I can tell they seem unused in this file